monix-catnap/shared/src/main/scala/monix/catnap/FutureLift.scala

```diff
@@ -59,8 +60,8 @@ import scala.concurrent.{Future => ScalaFuture}
   *   val sum: IO[Int] = IO(delayed(1 + 1)).futureLift
   * }}}
   */
-trait FutureLift[F[_], Future[_]] {
```

Comment:
A bit off-topic but it reminded me:

https://github.com/typelevel/cats-effect/pull/546

`Cats-Effect` changed `fromFuture` to do `shift`. Should we reconsider it on our part too?

Reply:
Yes, maybe.